README.md

```diff
@@ -100,4 +112,57 @@ terramate install-completions
 
 ### Project Setup
 
-TODO: More details on the basics to setup a project
+If you already have a Terraform project versioned on Git setting up
+Terramate is as easy as just [installing Terramate](#installing).
+Terramate comes with sensible defaults so just using it inside a pre existent
+Git repository should not require any configurations.
+
+The exception being repositories that have a default remote branch
+other than `origin/main`, in that case to make change detection work you will
+need to set a customized [project configuration](docs/project-config.md).
+
+If you want to play around with Terramate from scratch locally you can also
+setup a local git repository:
```

Comment:
just wondering if it's not easier and less troublesome if we just say they could create and clone an empty GH repository?

Reply:
I wondered the same ...that is why I also started by saying that if you already have a project versioned on Git it would be trivial to set Terramate up. But I'm not sure about this either. WDYT @mariux @soerenmartius ?